### ISSUE TITLE
Failed to load plugin with eslint_d

### ISSUE THREAD

**Issue by kkb0318** [mantoni/eslint_d.js#233]
Hi,

When I tried to edit the React program using neovim, I'm getting the error

```
Error: Failed to load plugin 'jsx-a11y' declared in '.eslintrc.json' : ENOENT: no such file or directory, open 'XXXX/node_modules/eslint-plugin-jsx-a11y/node_modules/@babel/runtime/helpers/interopRequireDefault.js'
```
This error occurs not only in jsx-a11y but also in another plugin.
The error is similar to issue https://github.com/mantoni/eslint_d.js/issues/221.　

I am using mason.nvim to download eslint_d (ver12.2.1).

<img width="792" alt="image" src="https://user-images.githubusercontent.com/48209784/209910539-45c4957d-9129-4b80-864d-80c34ed6b8c7.png">

.eslintrc.json

```
{
  "root": true,
  "env": {
    "browser": true,
    "es2022": true
  },
  "extends": [
    "eslint:recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:@typescript-eslint/recommended-requiring-type-checking",
    "standard-with-typescript",
    "plugin:jsx-a11y/recommended",
    "plugin:react/recommended",
    "plugin:react/jsx-runtime",
    "plugin:react-hooks/recommended",
    "prettier"
  ],
  "parserOptions": {
    "ecmaVersion": "latest",
    "tsconfigRootDir": ".",
    "project": ["./tsconfig.json"],
    "sourceType": "module"
  },
  "plugins": ["@typescript-eslint", "jsx-a11y", "react", "react-hooks"],
...
  }
}

```

I tried reinstalling eslint_d but the result was the same. 
Also, VSCode is able to load the eslint plugin, so I am wondering if it is an eslint_d error.

Best regards

**Comment by mantoni** [comment]
Did you try `eslint_d restart`? Reinstalling does not bounce the daemon process.

**Comment by kkb0318** [comment]
I did not know that deletion does not bounce the daemon process...


I couldn't find a way to restart with Mason, so once installed with npm, I ran the restart.

```
npm install eslint_d -g
eslint_d restart
```

After installing and being able to run it with command line, I did a restart, fixed the problem.

Thanks!! @mantoni
